coins/monero/src/wallet/mod.rs

```diff
@@ -218,4 +242,32 @@ impl Scanner {
       self.pair.view.deref() * spend,
     )
   }
+
+  /// Returns a featured address.
+  ///
+  /// if you created the scanner with `Some` value for `burning_bug` be aware that
+  /// even though you can pass `true` for the `guaranteed` parameter and generate
+  /// a guaranteed address, scanner wont be able decode incoming outputs for this address.
+  /// If you want to support guaranteed addresses please create a scanner with `None` value to
+  /// `burning_bug`. Reverse of the case also holds.
+  pub fn featured_address(
+    &mut self,
+    index: Option<(u32, u32)>,
```

Comment:
index -> subaddress? It's a clearer argument name, even though it would conflict with a variable in this function (which would then have to be renamed).